aspnetcore/grpc/protobuf.md

```diff
@@ -198,7 +198,11 @@ message DecimalValue {
 The `nanos` field represents values from `0.999_999_999` to `-0.999_999_999`. For example, the `decimal` value `1.5m` would be represented as `{ units = 1, nanos = 500_000_000 }`. This is why the `nanos` field in this example uses the `sfixed32` type, which encodes more efficiently than `int32` for larger values. If the `units` field is negative, the `nanos` field should also be negative.
 
 > [!NOTE]
-> There are multiple other algorithms for encoding `decimal` values as byte strings, but this message is easier to understand than any of them. The values are not affected by big-endian or little-endian on different platforms.
+> Additional algorithms are available for encoding `decimal` values as byte strings. The algorithm used by `DecimalValue`:
+> 
+> * Is easy to understand.
+> * Isn't affected by big-endian or little-endian on different platforms.
+> * Supports values in the range `9,223,372,036,854,775,808.999999999` to `9,223,372,036,854,775,807.999999999` with a maximum precision of nine decimal places, which isn't the full range of a `decimal`.
```

Comment:
@Liversage Should one of these values be negative? e.g.

```suggestion
> * Supports values in the range `9,223,372,036,854,775,808.999999999` to `-9,223,372,036,854,775,807.999999999` with a maximum precision of nine decimal places, which isn't the full range of a `decimal`.
```
Or the other way around.

Reply:
Ah! Sorry, but it's the other way around. The minimum value is `long.MinValue - 0.999_999_999M`.